cropper/src/main/java/com/canhub/cropper/CropImageView.kt

```diff
@@ -1088,15 +1088,16 @@ class CropImageView @JvmOverloads constructor(context: Context, attrs: Attribute
                 var uri = state.getParcelable<Uri>("LOADED_IMAGE_URI")
                 if (uri != null) {
                     val key = state.getString("LOADED_IMAGE_STATE_BITMAP_KEY")
-                    if (key != null) {
-                        val stateBitmap =
-                            if (BitmapUtils.mStateBitmap != null && BitmapUtils.mStateBitmap.first == key) BitmapUtils.mStateBitmap.second.get() else null
+                    key ?: run {
+                        val stateBitmap = BitmapUtils.mStateBitmap?.let {
+                            if (it.first == key) it.second.get() else null
+                        }
```

Comment:
The logic here changed.
Before `if key !=  null do stateBitmap...`
Now `if key == null do stateBitmap...`